src/App.jsx

```diff
@@ -0,0 +1,45 @@
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import Card from './Card';
+import Logo from './components/Logo';
+
+function App() {
+  const [datas, setDatas] = useState([]);
```

Comment:
always a good practice to name your state according to the nature of the data you are storing

Reply:
Alright, I will take note of that.